packages/misc/builder/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Build typeberry package into one CommonJS module",
   "type": "module",
   "scripts": {
-    "build": "rollup -c"
+    "build": "node --max-old-space-size=8192 ../../../node_modules/rollup/dist/bin/rollup -c"
```

Comment:
there should be a symlink in `node_modules/.bin` so it should be enough to do 
`node --max-old-space-size=8192 rollup -c`

or

`NODE_OPTIONS=--max-old-space-size=8192 rollup -c`

Reply:
thanks! Note the higher memory usage is only required to properly create the typings, not sure why that's the case. Previous approach of just importing `pkg.d.ts` was fast but the types were pretty incomplete, resulting in `any` type in random places.